spec/features/course/assessment/submission/log_spec.rb

```diff
@@ -85,8 +89,7 @@
 
         visit course_assessment_submission_logs_path(course, protected_assessment, submission)
 
-        expect(page).
-          to have_selector('h1', text: I18n.t('course.assessment.submission.logs.index.header'))
+        expect(page).to have_selector('div#submission-log')
```

Comment:
After you add the page header, check for the text

Reply:
done